web/themes/joinup/prototype/sass/components/_featured-lg.sass

```diff
@@ -61,6 +62,11 @@
       width: 200px
       margin-right: 40px
 
+    a
```

Comment:
![zrzut ekranu 2017-06-20 o 22 12 47](https://user-images.githubusercontent.com/10449853/27353818-c061013e-5605-11e7-8654-8f25b1d867e5.png)
It's overridden default link color for this attached items.